about.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Incoming Movies About</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="about.css">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Lato:wght@900&family=Poppins:wght@100;300&display=swap"
+    rel="stylesheet">
+</head>
+
+<body>
+  <header class="w-100 header-css about-page pt-5 pt-md-0 d-md-flex flex-md-column">
+    <div class="d-none header-top page-padding d-md-flex w-100 justify-content-end align-items-center">
+      <a href="#" class="me-2"><img class="header-icons" alt="facebook link"
+          src="content/images/facebook-svgrepo-com.svg"></a>
+      <a href="#" class="me-2"><img class="header-icons" alt="twitter link" src="content/images/twitter.svg"></a>
+      <a href="#" class="me-2 text-size font-secondary text-decoration-none color-thirthary">English</a>
+      <a href="#" class="me-2 text-size font-secondary text-decoration-none color-thirthary">My Page</a>
+      <a href="#" class="me-2 text-size font-secondary text-decoration-none color-thirthary">Logout</a>
+    </div>
```

Comment:
## [OPTIONAL]
- [ ] Would you kindly remove the padding top (`pt-5`) on your header so that your design should contain some blank space.

![image](https://user-images.githubusercontent.com/65068771/164846764-d20a2612-450f-4bf4-b109-1bb03d4e312f.png)